monkey/infection_monkey/utils/signal_handler.py

```diff
@@ -0,0 +1,30 @@
+import logging
+import signal
+
+from infection_monkey.i_master import IMaster
+from infection_monkey.utils.environment import is_windows_os
+from infection_monkey.utils.exceptions.planned_shutdown_exception import PlannedShutdownException
+
+logger = logging.getLogger(__name__)
+
+
+class StopSignalHandler:
+    def __init__(self, master: IMaster):
+        self._master = master
+
+    def __call__(self, signum, __=None):
```

Comment:
what's the point of specifying the default argument for `__`? Also, no point in using double underscore, we should use _.

Reply:
As a guideline I suggest using `_` for one unused argument and `*_` if there are many